dbclient/WorkspaceClient.py

```diff
@@ -388,22 +400,31 @@ def log_all_workspace_items(self, ws_path, workspace_log_writer, libs_log_writer
                 libs_log_writer.write(json.dumps(y) + '\n')
```

Comment:
We can also parallelize this ?

Reply:
No need to because they are only doing the writing part to the same file. (there is 0 performance improvement when it comes to writing to same files from multiple threads)